src/two/index.html

```diff
@@ -47,18 +47,17 @@
   </div>
   <div id="navcontainer">
     <a class="navtabtraccurrent" href="/two/">Home</a>
-    <a class="navtab" href="http://trac.openlayers.org">Support &amp; Development</a>
+    <a class="navtab" href="https://trac.openlayers.org">Support &amp; Development</a>
```

Comment:
We should probably update this URL.  Not a valid certificate for https://trac.openlayers.org (this comes from osgeo.org).